### PR TITLE
chore: update term data with winter 2025

### PR DESCRIPTION
**Pull request by KevinWu098**
## Summary
1. Add winter 2025
## Test Plan
1. tests don't fail

### CODE REVIEW

File: apps/antalmanac/src/lib/termData.ts
Comment (alexanderl19) [inline apps/antalmanac/src/lib/termData.ts]: Not sure what the finals date is used for, but comment on L28 mentions it should be the first day (sat). This would correspond to Mar 15 (`[2025, 2, 15]`).

**Review by alexanderl19 — CHANGES_REQUESTED** [review]

File: apps/antalmanac/src/lib/termData.ts
Comment (alexanderl19) [reply apps/antalmanac/src/lib/termData.ts]: Feel free to dismiss my review to merge this PR if it's not important.

Comment (KevinWu098) [reply apps/antalmanac/src/lib/termData.ts]: smh no trust -- good catch xD

**Review by alexanderl19 — APPROVED** [review]

**Comment by alexanderl19** [comment]
Not sure what's blocking this PR from being merged. Feel free to ping me if you want me to override and merge anyways.

Edit: Someone was requesting this already, so I went ahead and force merged.

**Pull request merged**
